modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2NewKeyGenerationPortalPart2.testcase

```diff
@@ -43,10 +44,71 @@ definition {
 	}
 
     @description = "Automation ID: LPS-151788 | Test Summary: Verify if when user fill the mandatory field and at least one detail from Activation Key Server Details, button 'Generate 1 Key' is already enable"
+	@ignore = "true"
 	@priority = "4"
 	test IfGenerate1KeyIsEnable {
 
-        // TODO LPS-151788 CP2NewKeyGenerationPortalPart2 archive pending implementation
+		// Needs refactor since it depends on example NASA ICE account
+
+		task ("Given a user signs in on OKTA to update the Customer Portal") {
+			CustomerPortal.loginOktaToUpdateCP();
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoSite(key_site = "${testSiteName}");
+		}
+
+		task ("Given I go to the test account") {
```

Comment:
Hi @luanakarla, this task description should be "And I go to the test account" because it's a succession from the previous Given. That makes the BDD more fluidly structured by replacing the successive Given’s, When’s, or Then’s with And’s and But’s. You can read more about this [here](https://cucumber.io/docs/gherkin/reference/)

Reply:
This description is standard. It is present in almost all tests. Perhaps this change is not necessary.